com.retrodev.blastem.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "com.retrodev.blastem",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "blastem",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Joystick and GPU access */
+        "--device=all",
+        /* Get access to the files */
+        "--filesystem=home:ro",
+        "--filesystem=xdg-run/gvfs:ro"
+    ],
+    "modules" : [
+        "shared-modules/glu/glu-9.0.0.json",
+        "shared-modules/glew/glew.json",
+        {
+            "name" : "blastem",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "DATA_PATH=/app/share/blastem/ CONFIG_PATH=/app/etc/ make blastem",
+                "install -m0755 -D blastem /app/bin/blastem.bin",
+                "install -m0755 -D blastem.sh /app/bin/blastem",
+                "install -d /app/share/blastem/",
+                "cp -r shaders/ images/ gamecontrollerdb.txt rom.db /app/share/blastem/",
+                "install -m0644 -D default.cfg /app/etc/default.cfg",
+                "install -m0644 -D com.retrodev.blastem.png /app/share/icons/com.retrodev.blastem.png",
```

Comment:
Incorrect path?

Reply:
No, it's correct and exported. It's a path for icons that don't have multiple sizes, don't have sizes that correspond to the set icon sizes, or don't have scalable versions. This is also the path I use for the icons in Flash player and Teo.